docs/src/main/asciidoc/amazon-lambda.adoc

```diff
@@ -246,18 +247,84 @@ aws lambda update-function-code --function-name my-native-function \
 
 == Examine the POM
 
-If you want to adapt an existing project to use Quarku's Amazon Lambda extension, there's a couple
+If you want to adapt an existing project to use Quarkus's Amazon Lambda extension, there's a couple
 of things you need to do.  Take a look at the generated example project to get an example of what you need to adapt.
 
 1. Include the `quarkus-amazon-lambda` extension as a pom dependency
-2. Configure Quarkus to build an `uber-jar`
+2. Configure Quarkus to build an `uber-jar` (via quarkus.package.uber-jar=true in the application.properties)
 3. If you are doing a native GraalVM build, Amazon requires you to rename your executable to `bootstrap` and zip it up.  Notice that the `pom.xml` uses the `maven-assemby-plugin` to perform this requirement.
 
+NB: With gradle, to build the uber-jar execute: ./gradlew quarkusBuild --uber-jar
 
+== Testing with the SAM CLI
 
+The https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html[AWS SAM CLI] allows you to run your lambda's locally on your laptop in a simulated Lambda environment.  This requires https://www.docker.com/products/docker-desktop[docker] to be installed.
 
+With a simple `template.yaml` you can combine both the java and native versions of your lambda function.  Or if you prefer, you can seperate the `template.yaml` into two files, `sam.jvm.yaml` and `sam.native.yaml` as per the link:amazon-lambda-http.html[Lambda HTTP guide]
```

Comment:
change to "into two files:  `sam.jvm.yaml` and `sam.native.yaml`" ...

Reply:
Done, please review.